### ISSUE TITLE
Cloudflare captcha challenge presented for ww1.torrent9.nz (cfscrape cannot solve captchas). Your OpenSSL version is lower than 1.1.1. Please upgrade your OpenSSL library and recompile Python.

### ISSUE THREAD

**Issue by janusn** [SickChill/SickChill#5515]
### INFO
Python Version: **2.7.14 (default, May 28 2018, 14:47:18) [GCC 4.7.3 20130102 (prerelease)]**
Operating System: **Linux-4.4.59+-x86_64-with-glibc2.2.5**
Locale: None
Branch: **master**
Commit: SickChill/SickChill@86207e9e728f4685e5b92a14674d2ae4e6e1279b
No Log available with ERRORS:
### ERROR
This exact error was seen ~11 times:
```
SEARCHQUEUE-BACKLOG-75414 :: [Torrent9] :: [86207e9] Request failed: Cloudflare captcha challenge presented for ww1.torrent9.nz (cfscrape cannot solve captchas). Your OpenSSL version is lower than 1.1.1. Please upgrade your OpenSSL library and recompile Python. (<type 'str'>)
```
Another error:
```
FINDPROPERS :: [Torrent9] :: [86207e9] Request failed: Cloudflare captcha challenge presented for ww1.torrent9.nz (cfscrape cannot solve captchas). Your OpenSSL version is lower than 1.1.1. Please upgrade your OpenSSL library and recompile Python. (<type 'str'>)
```
---
_STAFF NOTIFIED_: @SickChill/owners @SickChill/moderators

**Comment by janusn** [comment]
### INFO
Python Version: **2.7.14 (default, May 28 2018, 14:47:18) [GCC 4.7.3 20130102 (prerelease)]**
Operating System: **Linux-4.4.59+-x86_64-with-glibc2.2.5**
Locale: None
Branch: **master**
Commit: SickChill/SickChill@86207e9e728f4685e5b92a14674d2ae4e6e1279b
No Log available with ERRORS:
### ERROR
```
FINDPROPERS :: [Torrent9] :: [86207e9] Request failed: Cloudflare captcha challenge presented for ww1.torrent9.nz (cfscrape cannot solve captchas). Your OpenSSL version is lower than 1.1.1. Please upgrade your OpenSSL library and recompile Python. (<type 'str'>)
```
---
_STAFF NOTIFIED_: @SickChill/owners @SickChill/moderators

**Comment by janusn** [comment]
### INFO
Python Version: **2.7.14 (default, May 28 2018, 14:47:18) [GCC 4.7.3 20130102 (prerelease)]**
Operating System: **Linux-4.4.59+-x86_64-with-glibc2.2.5**
Locale: None
Branch: **master**
Commit: SickChill/SickChill@86207e9e728f4685e5b92a14674d2ae4e6e1279b
No Log available with ERRORS:
### ERROR
```
SEARCHQUEUE-BACKLOG-313435 :: [Torrent9] :: [86207e9] Request failed: Cloudflare captcha challenge presented for ww1.torrent9.nz (cfscrape cannot solve captchas). Your OpenSSL version is lower than 1.1.1. Please upgrade your OpenSSL library and recompile Python. (<type 'str'>)
```
---
_STAFF NOTIFIED_: @SickChill/owners @SickChill/moderators

**Comment by janusn** [comment]
### INFO
Python Version: **2.7.14 (default, May 28 2018, 14:47:18) [GCC 4.7.3 20130102 (prerelease)]**
Operating System: **Linux-4.4.59+-x86_64-with-glibc2.2.5**
Locale: None
Branch: **master**
Commit: SickChill/SickChill@86207e9e728f4685e5b92a14674d2ae4e6e1279b
No Log available with ERRORS:
### ERROR
```
SEARCHQUEUE-BACKLOG-78916 :: [Torrent9] :: [86207e9] Request failed: Cloudflare captcha challenge presented for ww1.torrent9.nz (cfscrape cannot solve captchas). Your OpenSSL version is lower than 1.1.1. Please upgrade your OpenSSL library and recompile Python. (<type 'str'>)
```
---
_STAFF NOTIFIED_: @SickChill/owners @SickChill/moderators

**Comment by janusn** [comment]
### INFO
Python Version: **2.7.14 (default, May 28 2018, 14:47:18) [GCC 4.7.3 20130102 (prerelease)]**
Operating System: **Linux-4.4.59+-x86_64-with-glibc2.2.5**
Locale: None
Branch: **master**
Commit: SickChill/SickChill@86207e9e728f4685e5b92a14674d2ae4e6e1279b
No Log available with ERRORS:
### ERROR
```
SEARCHQUEUE-BACKLOG-137151 :: [Torrent9] :: [86207e9] Request failed: Cloudflare captcha challenge presented for ww1.torrent9.nz (cfscrape cannot solve captchas). Your OpenSSL version is lower than 1.1.1. Please upgrade your OpenSSL library and recompile Python. (<type 'str'>)
```
---
_STAFF NOTIFIED_: @SickChill/owners @SickChill/moderators

**Comment by janusn** [comment]
### INFO
Python Version: **2.7.14 (default, May 28 2018, 14:47:18) [GCC 4.7.3 20130102 (prerelease)]**
Operating System: **Linux-4.4.59+-x86_64-with-glibc2.2.5**
Locale: None
Branch: **master**
Commit: SickChill/SickChill@86207e9e728f4685e5b92a14674d2ae4e6e1279b
No Log available with ERRORS:
### ERROR
```
SEARCHQUEUE-BACKLOG-72493 :: [Torrent9] :: [86207e9] Request failed: Cloudflare captcha challenge presented for ww1.torrent9.nz (cfscrape cannot solve captchas). Your OpenSSL version is lower than 1.1.1. Please upgrade your OpenSSL library and recompile Python. (<type 'str'>)
```
---
_STAFF NOTIFIED_: @SickChill/owners @SickChill/moderators

**Comment by janusn** [comment]
### INFO
Python Version: **2.7.14 (default, May 28 2018, 14:47:18) [GCC 4.7.3 20130102 (prerelease)]**
Operating System: **Linux-4.4.59+-x86_64-with-glibc2.2.5**
Locale: None
Branch: **master**
Commit: SickChill/SickChill@86207e9e728f4685e5b92a14674d2ae4e6e1279b
No Log available with ERRORS:
### ERROR
```
SEARCHQUEUE-BACKLOG-142991 :: [Torrent9] :: [86207e9] Request failed: Cloudflare captcha challenge presented for ww1.torrent9.nz (cfscrape cannot solve captchas). Your OpenSSL version is lower than 1.1.1. Please upgrade your OpenSSL library and recompile Python. (<type 'str'>)
```
---
_STAFF NOTIFIED_: @SickChill/owners @SickChill/moderators

**Comment by janusn** [comment]
### INFO
Python Version: **2.7.14 (default, May 28 2018, 14:47:18) [GCC 4.7.3 20130102 (prerelease)]**
Operating System: **Linux-4.4.59+-x86_64-with-glibc2.2.5**
Locale: None
Branch: **master**
Commit: SickChill/SickChill@86207e9e728f4685e5b92a14674d2ae4e6e1279b
No Log available with ERRORS:
### ERROR
```
SEARCHQUEUE-BACKLOG-71535 :: [Torrent9] :: [86207e9] Request failed: Cloudflare captcha challenge presented for ww1.torrent9.nz (cfscrape cannot solve captchas). Your OpenSSL version is lower than 1.1.1. Please upgrade your OpenSSL library and recompile Python. (<type 'str'>)
```
---
_STAFF NOTIFIED_: @SickChill/owners @SickChill/moderators

**Comment by janusn** [comment]
### INFO
Python Version: **2.7.14 (default, May 28 2018, 14:47:18) [GCC 4.7.3 20130102 (prerelease)]**
Operating System: **Linux-4.4.59+-x86_64-with-glibc2.2.5**
Locale: None
Branch: **master**
Commit: SickChill/SickChill@86207e9e728f4685e5b92a14674d2ae4e6e1279b
No Log available with ERRORS:
### ERROR
```
SEARCHQUEUE-BACKLOG-259675 :: [Torrent9] :: [86207e9] Request failed: Cloudflare captcha challenge presented for ww1.torrent9.nz (cfscrape cannot solve captchas). Your OpenSSL version is lower than 1.1.1. Please upgrade your OpenSSL library and recompile Python. (<type 'str'>)
```
---
_STAFF NOTIFIED_: @SickChill/owners @SickChill/moderators

**Comment by janusn** [comment]
### INFO
Python Version: **2.7.14 (default, May 28 2018, 14:47:18) [GCC 4.7.3 20130102 (prerelease)]**
Operating System: **Linux-4.4.59+-x86_64-with-glibc2.2.5**
Locale: None
Branch: **master**
Commit: SickChill/SickChill@86207e9e728f4685e5b92a14674d2ae4e6e1279b
No Log available with ERRORS:
### ERROR
```
SEARCHQUEUE-BACKLOG-72352 :: [Torrent9] :: [86207e9] Request failed: Cloudflare captcha challenge presented for ww1.torrent9.nz (cfscrape cannot solve captchas). Your OpenSSL version is lower than 1.1.1. Please upgrade your OpenSSL library and recompile Python. (<type 'str'>)
```
---
_STAFF NOTIFIED_: @SickChill/owners @SickChill/moderators

**Comment by janusn** [comment]
### INFO
Python Version: **2.7.14 (default, May 28 2018, 14:47:18) [GCC 4.7.3 20130102 (prerelease)]**
Operating System: **Linux-4.4.59+-x86_64-with-glibc2.2.5**
Locale: None
Branch: **master**
Commit: SickChill/SickChill@86207e9e728f4685e5b92a14674d2ae4e6e1279b
No Log available with ERRORS:
### ERROR
```
SEARCHQUEUE-BACKLOG-78922 :: [Torrent9] :: [86207e9] Request failed: Cloudflare captcha challenge presented for ww1.torrent9.nz (cfscrape cannot solve captchas). Your OpenSSL version is lower than 1.1.1. Please upgrade your OpenSSL library and recompile Python. (<type 'str'>)
```
---
_STAFF NOTIFIED_: @SickChill/owners @SickChill/moderators

**Comment by janusn** [comment]
### INFO
Python Version: **2.7.14 (default, May 28 2018, 14:47:18) [GCC 4.7.3 20130102 (prerelease)]**
Operating System: **Linux-4.4.59+-x86_64-with-glibc2.2.5**
Locale: None
Branch: **master**
Commit: SickChill/SickChill@86207e9e728f4685e5b92a14674d2ae4e6e1279b
No Log available with ERRORS:
### ERROR
```
SEARCHQUEUE-BACKLOG-120151 :: [Torrent9] :: [86207e9] Request failed: Cloudflare captcha challenge presented for ww1.torrent9.nz (cfscrape cannot solve captchas). Your OpenSSL version is lower than 1.1.1. Please upgrade your OpenSSL library and recompile Python. (<type 'str'>)
```
---
_STAFF NOTIFIED_: @SickChill/owners @SickChill/moderators

**Comment by pro-src** [comment]
Hi @janusn,

Cloudflare is always improving their Anti-Scrape shield. I'd like to help resolve this issue but I'm going to need more information and possibly for you to run some tests.

Since this issue is with cloudflare-scrape failing to bypass Cloudflare's shield, please open an issue at https://github.com/Anorov/cloudflare-scrape. I will personally help you with this problem when you do.

Do you mind cleaning up this issue a bit by removing the extra comments? I noted the duplicates in your original post.

Thanks.

**Comment by pro-src** [comment]
@janusn I've created an issue at https://github.com/Anorov/cloudflare-scrape/issues/259
Please follow that issue.

**Comment by janusn** [comment]
@pro-src Thanks for taking up this issue. I've just deleted the duplicated comments. I reported the errors by the report button on the UI which caused duplications.

It is midnight on my time zone. Please let me know how to fix it and tests. I will help you tomorrow.

Thank you!

**Comment by pro-src** [comment]
I'm currently looking for a solid solution to this problem but if all else fails, I do know of a way to avoid this error. So we'll get you back up and running with torrent9 for sure.

You're welcome :smiley:

**Comment by WebSpider** [comment]
Feel free to PR either SC or upstream :sunglasses:

**Comment by pro-src** [comment]
Fixed by #5516 and is on the master branch.
